dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/BaseServiceImpl.java

```diff
@@ -122,6 +120,24 @@ public void putMsg(Result result, Status status, Object... statusParams) {
         }
     }
 
+    /**
+     * put Map object with both status and message to result object
+     *
+     * @param result result code
+     * @param mapObj Map object contain status and message.
+     */
+    @Override
+    public void putMsg(Result<Object> result, Map<String, Object> mapObj) {
```

Comment:
Add for convenience method, I add this function more easier to convert map from `valid` to result